specutils/analysis/template_comparison.py

```diff
@@ -160,3 +183,54 @@ def template_match(observed_spectrum, spectral_templates,
             smallest_chi_index = index
 
     return smallest_chi_spec, chi2_min, smallest_chi_index
+
+
+def template_redshift(observed_spectrum, template_spectrum, min_redshift, max_redshift, delta_redshift):
+    """
+    Find the most accurate redshift for template_spectrum to match observed_spectrum using chi2
+
+    Parameters
+    ----------
+    observed_spectrum : :class:`~specutils.Spectrum1D`
+        The observed spectrum.
+    template_spectrum : :class:`~specutils.Spectrum1D`
+        The template spectrum, which will have it's redshift calculated
+    min_redshift : `float`
+        The minimum redshift allowed
+    max_redshift : `float`
+        The maximum redshift allowed
+    delta_redshift : `float`
+        The amount the redshift will change between loops
+
+    Returns
+    -------
+    final_redshift : `float`
+        The most accurate redshift for template_spectrum to match the observed_spectrum
```

Comment:
```suggestion
        The most best-fit redshift for template_spectrum to match the observed_spectrum
```
To clairfy the reasoning behind this suggestion: "most accurate" has a subtly-different meaning here - "most accurate" implies "the redshift closest to the true redshift", which is not quite the same as "the redshift closest to the redshift of the best-fit template".

(Also, in my experience, this is the words scientists actually use for this... which is perhaps more improtant than anything else?  ¯\_(ツ)_/¯)